lib/methods/core/getTransaction/GetTransactionResponse.js

```diff
@@ -3,7 +3,7 @@ const InvalidResponseError = require('../../platform/response/errors/InvalidResp
 class GetTransactionResponse {
   /**
    *
-   * @param {Object} properties
+   * @param {object} properties
```

Comment:
Linter recommends using `Object`, isn't it?

Reply:
No
This was fixed by eslint --fix